_pages/coc.md

```diff
@@ -74,7 +80,17 @@ communications pertaining to community business.
 
 ## Contact Us ##
 
-We won't share your contact information or the contents of your message with anyone other than conference executives.
+Note that Evy Kassirer and Arshia Mufti are StarCon's official incident responders, and Evy has participated in a Code of Conduct enforcement training with [Sage Sharp](http://sage.thesharps.us/). You are welcome to reach out to them or to any other conference organizers or volunteers.
+
+You have several options for contacting us:
+
+- Send a private Slack message to any [conference organizer](/#whos-running-this)
+- Direct message [Evy](https://twitter.com/EvyKassirer) or [Arshia](https://twitter.com/arshia__) on Twitter
+- Message Evy securely on [Keybase](https://keybase.io/evyk/)
```

Comment:
arshia do you want to also get keybase?

Reply:
here you go: https://keybase.io/arshiamufti